lib/SPIRV/libSPIRV/SPIRVType.h

```diff
@@ -1081,11 +1077,20 @@ class SPIRVTypeJointMatrixINTEL : public SPIRVType {
   SPIRVCapVec getRequiredCapability() const override {
     return {internal::CapabilityJointMatrixINTEL};
   }
+  void setWordCount(SPIRVWord WordCount) override {
+    SPIRVType::setWordCount(WordCount);
+    Args.resize(WordCount - FixedWC);
+  }
   SPIRVType *getCompType() const { return CompType; }
-  SPIRVValue *getLayout() const { return Layout; }
-  SPIRVValue *getRows() const { return Rows; }
-  SPIRVValue *getColumns() const { return Columns; }
-  SPIRVValue *getScope() const { return Scope; }
+  SPIRVValue *getRows() const { return Args[0]; }
+  SPIRVValue *getColumns() const { return Args[1]; }
+  SPIRVValue *getLayout() const { return Args[2]; }
+  SPIRVValue *getScope() const { return Args[3]; }
+  SPIRVValue *getUse() const {
+    if (Args.size() > 4)
```

Comment:
You can ignore it. But do you mind using: `return Args.size() > 4 ?  return Args[4] : nullptr`

Reply:
Thanks! Applied